iot-devices/platformio.ini

```
@@ -11,6 +11,8 @@ lib_deps =
   bblanchon/ArduinoJson@^7.0.4
   adafruit/Adafruit SSD1306@^2.5.10
   adafruit/Adafruit GFX Library@^1.11.9
+  # paulstoffregen/OneWire@^2.3.8
+  # milesburton/DallasTemperature@^3.11.0
```

I think we need to add TODO comment for this:


```suggestion
TODO: uncomment this once the temperature sensor is availabe
  # paulstoffregen/OneWire@^2.3.8
  # milesburton/DallasTemperature@^3.11.0
```

---

updated in updated in 8256344e5270c9c577e33f91f681641a22bc5cb3